odbc-api/src/connection.rs

```diff
@@ -240,6 +240,11 @@ impl<'c> Connection<'c> {
     pub unsafe fn promote_to_send(self) -> force_send_sync::Send<Self> {
         force_send_sync::Send::new(self)
     }
+
+    /// Get the name of the database management system used by the connection.
+    pub fn database_management_system_name(&self, name: &mut OutputStringBuffer) -> Result<(), Error> {
```

Comment:
With regards to `OutputStringBuffer` see the comment below. Other than that it would be nice to have a convinience method directly returning `String`. I am not quite sure how to name the method taking the output buffer vs the method returning the `String`.

`fetch_database_managment_system_name` vs `database_managment_system_name`?